ledger/light/executor/request_checker.go

```diff
@@ -71,6 +71,13 @@ func (c *RequestCheckerDefault) CheckRequest(ctx context.Context, requestID inso
 	reasonRef := request.ReasonRef()
 	reasonID := *reasonRef.GetLocal()
 
+	if reasonID.IsEmpty() {
```

Comment:
Isn't it the same check in insolar/record/record.go:205 ?
`r.ReasonRef().IsEmpty()`